azavea/tilegarden
# Feature/travis ci

## Overview
This PR adds Travis to the project, which I've configured to run tests on PRs.


### Demo
![screen shot 2018-07-11 at 14 36 21-fullpage](https://user-images.githubusercontent.com/16990679/42592542-d85b5306-8517-11e8-9b12-41d155d37b71.png)


## Testing Instructions

 * Visit https://travis-ci.org/azavea/tilegarden
 * Under "settings" on the right side of the screen, click "trigger build", and trigger one on this branch.
 * Alternatively, this pull request should just automatically trigger a Travis build, and GitHub should display the build's status.


Resolves #15
